patches/api/0459-Player-standing-on-position-API.patch

```diff
@@ -0,0 +1,41 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Owen1212055 <23108066+Owen1212055@users.noreply.github.com>
+Date: Fri, 9 Jun 2023 22:49:45 -0400
+Subject: [PATCH] Player standing on position API
+
+
+diff --git a/src/main/java/org/bukkit/entity/Entity.java b/src/main/java/org/bukkit/entity/Entity.java
+index 6ace3581f8d0c2a1b7e2188d5b6af5c984b74a0e..b366bc329fc8714b4cb2e129f97bf9569ab9ade0 100644
+--- a/src/main/java/org/bukkit/entity/Entity.java
++++ b/src/main/java/org/bukkit/entity/Entity.java
+@@ -1107,4 +1107,30 @@ public interface Entity extends Metadatable, CommandSender, Nameable, Persistent
+      */
+     @NotNull String getScoreboardEntryName();
+     // Paper end - entity scoreboard name
++    // Paper start - Player standing on position API
++    /**
++     * Gets the block that is currently being used to calculate movement effects (i.e friction) on the entity.
++     * <p>
++     * Examples:
++     * - When standing on a slab under an ice block, this will return the position of the ice block.
```

Comment:
This reads wierdly, like if the player stands on the ice and under the ice is a slab.

```suggestion
+     * - When standing on a slab on top of an ice block, this will return the position of the ice block.
```